kong/clustering/utils.lua

```diff
@@ -185,12 +184,15 @@ else
     return res, err
   end
 
+  _M.json_encode = cjson.encode
+  --[[ TODO: fix the behavior of simdjson encoding
```

Comment:
I wouldn't say this is something we need to fix, let's jut explain the different behavior in this TODO section.

Reply:
We should fix this in the library eventually. The reason why I’m asking the encoding behavior being consistent instead of document as a difference is because we may use this new library in more places in the future. In this case, introducing this kind of behavior difference and being opinionated does us almost no good and should be avoided.

Reply:
changed.

Reply:
This is not something we should fix in the Gateway repo, yes, we should fix it in the library.